hledger-lib/Hledger/Utils/Text.hs

```diff
@@ -207,71 +194,6 @@ textConcatBottomPadded :: [Text] -> Text
 textConcatBottomPadded = TL.toStrict . renderRow def{tableBorders=False, borderSpaces=False}
                        . Group NoLine . map (Header . textCell TopLeft)
 
--- -- | Join multi-line strings horizontally, after compressing each of
--- -- them to a single line with a comma and space between each original line.
--- concatOneLine :: [String] -> String
--- concatOneLine strs = concat $ map ((intercalate ", ").lines) strs
```

Comment:
These were reminders+templates for doing a full port of this collection of text utilities, on the theory that they might be useful to others even if our usage fluctuates, but it hasn't happened yet, ok.